x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -579,8 +579,8 @@ public void testWhereOnNested() {
         accept("SELECT int FROM test WHERE dep.start_date > '2020-01-30'::date AND (int > 10 OR dep.end_date IS NULL) " +
                "OR NOT(dep.start_date >= '2020-01-01')");
         String operator = randomFrom("<", "<=");
-        assertEquals("1:42: WHERE isn't (yet) compatible with scalar functions on nested fields [dep.location]",
-            error("SELECT shape FROM test " +
+        assertEquals("1:46: WHERE isn't (yet) compatible with scalar functions on nested fields [dep.location]",
+            error("SELECT geo_shape FROM test " +
```

Comment:
Any reason for these changes?  Guessing that we'd change, but not extend the coverage by it?

Reply:
Added a field with the type and the name `shape` to increase coverage, so I renamed the existing `shape` field that has type `geo_shape` to `geo_shape` to avoid the name collision and make sure the name reflects the actual type of the field.